recipes/unitytest/all/test_v1_package/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+cmake_minimum_required(VERSION 3.1)
```

Comment:
The `test_v1_package` folder can be remove. It's no longer needed for new recipes: https://github.com/conan-io/conan-center-index/discussions/17406